std-bits/table/src/main/java/org/enso/table/read/DelimitedReader.java

```diff
@@ -490,9 +491,9 @@ private void markUsed() {
   private static final int INITIAL_ROW_CAPACITY = 100;
 
   private void initBuilders(int count) {
-    builders = new Builder[count];
+    builders = new ArrayList<>(count);
     for (int i = 0; i < count; i++) {
-      builders[i] = Builder.getForType(TextType.VARIABLE_LENGTH, INITIAL_ROW_CAPACITY, null);
+      builders.add(Builder.getForText(INITIAL_ROW_CAPACITY, TextType.VARIABLE_LENGTH));
```

Comment:
This makes me realize - should the `Builder.getForText` accept a problem aggregator?

I appreciate it is not using one now, but in future implementations - I guess it may. If we restrict ourselves to API that does not require the user to pass the aggregator - that may mean that if we want to introduce it in the future, that may come at an increased cost as we'd need to update the relevant usages.

OTOH I guess it is not that likely we'll need one and perhaps it's better to face such _hypothetical_ increased cost instead of making this unnecessarily complicated right now.